deploy/openshift-ci.sh

```diff
@@ -7,3 +7,5 @@ echo "Running out of ${DIR}"
 $DIR/install-openshift-pipelines.sh
 oc apply -f $DIR/namespace.yaml
 $DIR/patch-yaml.sh
+oc apply -k $DIR/overlays/development
```

Comment:
Are you sure you want this bit? A lot of the 'dev' overlay stuff is also setup in the test itself. e.g. this config map creation: https://github.com/redhat-appstudio/jvm-build-service/blob/857389d1fd39350509801fc67751211e9dd00d8e/openshift-with-appstudio-test/e2e/util.go#L202

The dev overlay also includes its own version, so if you are using the same namespace the test will fail.

Reply:
yeah the creation of an extra config map in the `test-jvm-namespace` namespace the e2e's will not use seemed like a small cost compared to being able to reuse the dev flow as much as possible and get the image substitution 

Not sure what you mean by the dev overlay using its own version, but the e2e's will not use `test-jvm-namespace` at all, and use the namesapce that configmap you referenced is in.

Now, if you think that calling 

`oc apply -k $DIR/operator/overlays/development` is sufficient for our means, where the overrides for the controller, sidecar, and req processor get overridden (I think I can see where that is the case, as I think the other images are referenced by the controller, right?), then sure, I'll test that out on Friday and switch to that.

Reply:
I think you would still need the patches on the maven task. I guess this is fine.

Reply:
forgot also add the crds and base/rbac as well via oc apply -f